optimum/onnxruntime/trainer.py

```diff
@@ -907,7 +907,7 @@ def evaluate(
         """
         # memory metrics - must set up as early as possible
         # TODO: We need to enable evaluation using ORT backend.
-        self.model = unwrap_model(self.model)
```

Comment:
Another question is, why only t5 model evaluation fails, but other models are good to evaluate?

Reply:
Based on the assumption that T5 is a sequence-to-sequence (seq2seq) model, the reason for its failure can be attributed to the fact that when the trainer falls back to the Seq2SeqTrainer class, it encounters an error.